src/app/medInria/main.cpp

```diff
@@ -270,6 +271,8 @@ int main(int argc,char* argv[])
 
     forceShow(*mainwindow);
 
+    med::python::setup();
```

Comment:
où est la déinitialisation de python

Reply:
La fonction `setup` connecte le fonction `teardown` au signal `aboutToQuit`. J'ai suivi les recommandations de Qt:

> We recommend that you connect clean-up code to the aboutToQuit() signal, instead of putting it in your application's main() function. This is because, on some platforms the QApplication::exec() call may not return. For example, on the Windows platform, when the user logs off, the system terminates the process after Qt closes all top-level windows. Hence, there is no guarantee that the application will have time to exit its event loop and execute code at the end of the main() function, after the QApplication::exec() call.

Reply:
nice